src/components/WorkOrder/WorkOrderHeader.tsx

```diff
@@ -77,9 +83,12 @@ const WorkOrderHeader = (props: Props) => {
         {'followOnRequest' in workOrder &&
           workOrder.followOnRequest !== null && <FurtherWorkRequiredFlag />}
 
-        {workOrder.operatives.length > 0 &&
-          ((workOrder.appointment && workOrder.appointmentISODatePassed()) ||
-            readOnly) && <Operatives operatives={workOrder.operatives} />}
+        {appointmentDetails.operatives.length > 0 &&
+          ((appointmentDetails.appointment &&
+            appointmentDetails.appointmentISODatePassed()) ||
+            readOnly) && (
+            <Operatives operatives={appointmentDetails.operatives} />
+          )}
```

Comment:
Operatives is now stored in appointmentDetails, so it is updated here (and other places)